scio-extra/src/main/scala/com/spotify/scio/extra/bigquery/syntax/SCollectionSyntax.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra.bigquery.syntax
+
+import com.google.api.services.bigquery.model.TableReference
+import com.spotify.scio.annotations.experimental
+import com.spotify.scio.bigquery.BigQueryTable.WriteParam
+import com.spotify.scio.bigquery.{BigQueryTable, Table, TableRow}
+import com.spotify.scio.io.ClosedTap
+import com.spotify.scio.util.ScioUtil
+import com.spotify.scio.values.SCollection
+import org.apache.avro.Schema
+import org.apache.avro.generic.IndexedRecord
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO.Write.{CreateDisposition, WriteDisposition}
+
+import scala.reflect.ClassTag
+
+trait SCollectionSyntax {
+  implicit def toAvroToBigQuerySCollection[T <: IndexedRecord: ClassTag](
+    data: SCollection[T]
+  ): AvroToBigQuerySCollection[T] = new AvroToBigQuerySCollection[T](data)
+}
+
+final class AvroToBigQuerySCollection[T <: IndexedRecord: ClassTag](
```

Comment:
🎨

```suggestion
final class AvroToBigQuerySCollectionOps[T <: IndexedRecord: ClassTag](
```

most of these are normally suffixed with `Ops`

Reply:
👍